plugins/orchestrator/src/components/WorkflowEditor/WorkflowEditor.tsx

```diff
@@ -217,10 +217,10 @@ const RefForwardingWorkflowEditor: ForwardRefRenderFunction<
             if (canceled.get()) {
               return;
             }
-            const definition = fromWorkflowSource(source);
+            const definition = fromWorkflowSource(source.data);
             setWorkflowDefinitionPromise({ data: definition });
 
-            const workflowFormat = extractWorkflowFormat(source);
+            const workflowFormat = extractWorkflowFormat(source.data);
```

Comment:
Something seems to not work here
With this PR when attempting to view a workflow with json format, such as Assessment Workflow from dev env, it navigates to /orchestrator/workflows/yaml/assessment instead of to  /orchestrator/workflows/json/assessment
This breaks the workflow viewer in this case

Reply:
The result of the API should be a string both for json and for yaml workflows
I noticed in the network tab that with main branch you see this in the response tab of the call to /source
![image](https://github.com/user-attachments/assets/7809d29e-0a7f-43cd-8c32-c29deb35c2f7)
But with this PR it's empty

Reply:
You right! 
Fixed here

See the results for: 
- Assessment
![image](https://github.com/user-attachments/assets/617acfa7-dc56-49a1-a111-714633db9951)

- Greeting yaml
![image](https://github.com/user-attachments/assets/c5b793da-2731-4575-be19-03b6c5c8d29d)